aten/src/ATen/native/tags.yaml

```diff
@@ -46,6 +46,15 @@
   desc: |
           This tag indicates that the operator should be passed Tensors following
           the same stride permutation as observed in eager when compiled in inductor.
+          Only one of {needs_fixed_stride_order, flexible_layout} can apply; if
```

Comment:
Does this PR only focus on `fixed_stride_order`? Do we need another `needs_exact_stride`?

Reply:
We need to add a needs_exact_stride at some point. It would be better for the default to be needs_exact_stride. This is a larger project though and needs some things we don't have yet

Reply:
it makes sense.